elections/election-2022.md

```diff
@@ -0,0 +1,115 @@
+# Project nomination for BSC representatives 2022
+
+In accordance with the [Elections Schedule](./schedule.md) and the
+[eBPF Foundation Charter][charter], this page holds a public record of the
+steps undertaken to nominate members for the BSC in 2022.
+
+The following members' terms will expire in 2022. The Section 4.b column
+indicates which projects must be considered to draw nominations from.
+
+| Name    | Section 4.b  |
+| ------- | ------------ |
+| (empty) | eBPF runtime |
+| TBD     |              |
+
+## Nominations (ETA August)
+
+The following nominees are under consideration for election to the BSC.
+
+### Kernel Candidates
+
+_[eBPF Foundation Charter][charter] §4.b.i Two Linux Kernel representatives
+from the group of eBPF Linux kernel maintainers (as specified in the Linux eBPF
+maintainer file). The Linux Kernel representatives must not be employed by the
+same Company or by Related Companies._
+
+* TBD
+
+### Runtime Candidates
+
+_[eBPF Foundation Charter][charter] §4.b.ii Up to two representatives from
+other open source eBPF runtime implementations.￼Each “eBPF Runtime
+Representative” will represent a different eBPF runtime chosen by the BSC, and
+must be an active contributor to that eBPF runtime._
+
+* eBPF Runtime for Windows
+* hBPF
+
+### Technical Project Candidates
+
+_[eBPF Foundation Charter][charter] §4.b.iii Up to three technical project
+representatives. Each Technical Project Representative candidate must be an
+active contributor to a different Technical Project chosen by the BSC._
+
+* TBD
+
+### Additional Project Candidates
+
+_[eBPF Foundation Charter][charter] §4.b.iv Up to two eBPF Landscape Project
+representatives. Each eBPF Landscape Project Representative candidate must be
+an active contributor to a different eBPF Landscape Project chosen by the
+BSC.._
+
+#### Major Applications
+* BCC[^1]
+* Cilium
+* bpftrace[^1]
+* Falco
```

Comment:
Actually as noted in https://github.com/ebpf-io/ebpf.io-website/issues/279, Brendan argues that Falco might fail to meet the requirement "The project must be using eBPF as its underlying core technology, in other words, a project would lose its purpose if the eBPF parts are removed" since it can supposedly function without ebpf.